apps/website/src/components/Placeholder.astro

```diff
@@ -0,0 +1,37 @@
+---
+import InformationIcon from '~/icons/InformationIcon.astro';
+
+export type Props = {
+  /** ending of the url for a specific storybook page */
+  componentPageName: string;
+};
+
+const { componentPageName, ...rest } = Astro.props;
+
+const storyUrl = `https://itwin.github.io/iTwinUI/react/?path=/story/${componentPageName}`;
+---
+
+<div {...rest}>
+  <InformationIcon aria-hidden='true' />
+  <p>
+    This page has not yet been finished and is being worked on. In the meantime, please visit
+    <a href={storyUrl}>storybook</a> to see more examples.
+  </p>
+</div>
+
+<style lang='scss'>
+  div {
+    display: flex;
+    align-items: center;
+    gap: var(--space-3);
+    padding: var(--space-3);
+    border: solid 1px #adb7f5;
+    border-radius: var(--border-radius-1);
+
+    svg {
+      color: #adb7f5;
```

Comment:
can you use one of the color variables defined in `_global.astro` (or add a new one if you think it makes sense)?

Reply:
Updated